config/v1/types_infrastructure.go

```diff
@@ -614,6 +626,14 @@ type OpenStackPlatformStatus struct {
 	// datacenter DNS, a DNS service is hosted as a static pod to serve those hostnames
 	// to the nodes in the cluster.
 	NodeDNSIP string `json:"nodeDNSIP,omitempty"`
+
+	// loadBalancerType defines if a internal or external load balancer
+	// is to be used for the openshift api and ingress endpoints.
+	// When set to Internal the static pods in charge of API and Ingress traffic load-balancing
+	// defined in machine config operator will be deployed.
+	// When set to External these static pods will not be deployed.
+	// +optional
```

Comment:
You can enforce the options here with (note the empty string for the default case):
```
// +kubebuilder:validation:Enum:="";"Internal";"External"
// +kubebuilder:validation:Required
```

Reply:
Yes but that means we need to repeat this validation when this will be defined in other Platforms, while now it's defined at the type level. I'm not sure if it's good practise or not, it's just something I saw elsewhere in the API.

2 other questions:
* Why would we support an empty string if the default is `Internal` ?
* Why would it be required?

Reply:
Please also check the policy on capitalised options; if you ask me, lowercase is less surprising

Reply:
I followed what Joel said and add the `PascalCase` convention.